pkgs/development/ocaml-modules/janestreet/janePackage.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, ocaml, jbuilder, findlib }:
+
+{ name, version ? "0.9.0", buildInputs ? [], hash, ...}@args:
+
+if !stdenv.lib.versionAtLeast ocaml.version "4.03"
+then throw "${name}-${version} is not available for OCaml ${ocaml.version}" else
+
+stdenv.mkDerivation (args // {
```

Comment:
`args // {` decreases overridability, normally one wants to do `} // args` at the end.